crates/proof-of-sql/src/sql/ast/inequality_expr.rs

```diff
@@ -22,12 +22,20 @@ pub struct InequalityExpr<C: Commitment> {
     lhs: Box<ProvableExprPlan<C>>,
     rhs: Box<ProvableExprPlan<C>>,
     is_lte: bool,
+    #[cfg(test)]
```

Comment:
Please add comments for it is not clear what `zero_sign` means. Alternatively something like `treat_zeros_as_negative` is more descriptive.